streaming/src/main/scala/org/apache/spark/streaming/util/StateMap.scala

```diff
@@ -77,6 +80,7 @@ private[streaming] class EmptyStateMap[K: ClassTag, S: ClassTag] extends StateMa
 }
 
 /** Implementation of StateMap based on Spark's [[org.apache.spark.util.collection.OpenHashMap]] */
+@DefaultSerializer(classOf[KryoJavaSerializer])
```

Comment:
Actually, this is not a correct fix because the user can put objects that only support Kryo into this Map. KryoJavaSerializer cannot handle them. Fixed this issue in my latest commit.